test/extended/openstack/loadbalancers.go

```diff
@@ -574,18 +576,23 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 		e2e.Logf("Removed LoadBalancerSourceRanges spec from the service")
 
 		// Wait until allowed_cidrs in the LB listener is updated to 0.0.0.0/0 (all traffic allowed)
-		allowAllAllowedCidrs := []string{"0.0.0.0/0"}
-		e2e.Logf("Expected allowed_cidrs: '%v'", allowAllAllowedCidrs)
-		o.Eventually(func() []string {
-			lbListener, err := octavialisteners.Get(loadBalancerClient, listenerId).Extract()
-			if err != nil {
-				e2e.Logf("Error ocurred: %v, trying next iteration", err)
-				return []string{}
-			}
-			e2e.Logf("Found AllowedCIDRs: %v", lbListener.AllowedCIDRs)
-			return lbListener.AllowedCIDRs
-		}, "10s", "1s").Should(o.ConsistOf(allowAllAllowedCidrs), "Didn't find the expected allowed_cidrs '%v'", allowAllAllowedCidrs)
-		e2e.Logf("Found expected allowed_cidrs '%v' in Openstack LoadBalancer Listener '%s'", allowAllAllowedCidrs, listenerId)
+		if lbProvider == "amphora" { // it only makes sense with Amphora
+			allowAllAllowedCidrs := []string{"0.0.0.0/0"}
+			e2e.Logf("Expected allowed_cidrs: '%v'", allowAllAllowedCidrs)
+			o.Eventually(func() []string {
+				lbListener, err := octavialisteners.Get(loadBalancerClient, listenerId).Extract()
+				if err != nil {
+					e2e.Logf("Error ocurred: %v, trying next iteration", err)
+					return []string{}
+				}
+				e2e.Logf("Found AllowedCIDRs: %v", lbListener.AllowedCIDRs)
+				return lbListener.AllowedCIDRs
+			}, "10s", "1s").Should(o.ConsistOf(allowAllAllowedCidrs), "Didn't find the expected allowed_cidrs '%v'", allowAllAllowedCidrs)
+			e2e.Logf("Found expected allowed_cidrs '%v' in Openstack LoadBalancer Listener '%s'", allowAllAllowedCidrs, listenerId)
+		} else {
+			// It should be fast with OVN, but no good way to check it, let's just sleep.
+			time.Sleep(30 * time.Second)
```

Comment:
can we maybe monitor the SG rules here as an active wait?

Reply:
Yeah, but it's not that simple to get that SG. We could use name to get it but I really want to avoid this, it's totally possible we'll be changing names soon.

Honestly I don't like checking `allowed_cidrs` either, we're checking something implementation-specific. It would be better to just try to connect until it suceeds.